website/static/js/fangorn.js

```diff
@@ -1009,6 +1020,34 @@ function _removeEvent (event, items, col) {
     }
 }
 
+function doCheckout(item, checker, error_message) {
+    $.ajax({
```

Comment:
After further investigation, it looks like you should be able to use `$osf.putJSON` here as long as you set `isCors: true`. `putJSON` calls `ajaxJSON`, which I believe is the preferred way to send requests to APIv2.